skycultures/balinese/description.md

```diff
@@ -0,0 +1,204 @@
+
+### Sky
+
+This sky culture has one small lintang system called "Kartika". Please zoom into the Pleiades to see it clearly. 
+Balinese people are gifted in art. They use art to admire the cosmos and the gods who guard it. Consequently, the constellations depicted in this culture are found not only on palm-leaf manuscripts but also on textiles sold in the marketplace. Furthermore, they also depict them on buildings such as the Floating Pavilion, used for royal gatherings, receiving guests, and ceremonial purposes.
+
+### Moon and Sun
+
+Bali has two distinct calendars: the Pawukon calendar and the Balinese Saka calendar.
+The Pawukon calendar is a purely arithmetic calendar with a 210-day cycle, divided into 30 seven-day units called wuku. This calendar does not have a new year but begins with the Sinta wuku. This calendar is related to the Palelintangan system. Furthermore, most feast days in Bali are determined based on the Pawukon calendar.
+Next is the Balinese Saka calendar, a Balinese Hindu lunisolar calendar with a normal year consisting of 12 months. The beginning of each month is determined by the new moon (called Tilem). The waxing phase of the moon is called Penanggal, and the waning phase is called Panglong. The difference in the number of days in the solar and lunar years results in an intercalation of 11 days, which is then added to form the 13th month every three years. The New Year is called Nyepi (Silence Day), the first day of the 10th month (Sasi Kadasa). It always falls on the day after Tilem, which is closest to (but after) the March equinox. Coupled with intercalation regulations, Nyepi is kept in the mid-March to mid-April Gregorian calendar.
+For the civil calendar, in Indonesia, all regions are required to follow the national calendar. However, Bali has its calendars for religious rituals. The Balinese government eventually took the initiative to combine various calendar systems and adapt all Indonesian holidays to the one civil Balinese calendar. As a result, this calendar circulating in the community has become a complex one, consisting of the Saka Balinese calendar, the Pawukon calendar, the Saka Javanese calendar, the Chinese calendar, and the Japanese calendar. Therefore, a single date contains information from various calendar systems used in Bali.
+
+### Other Celestial Objects
+
+The Balinese consider comets to be stars, calling them lintang Dupa. Furthermore, the lunar ascending and descending nodes are also considered stars, lintang Laweyan and lintang Děpat.
+
+## Constellations
+
+##### Kala Sungsang
+Kala Sungsang is an upside-down demon. Sungsang itself is inverted. Kala also means time. So the duty of this demon is to remind people who forget or even neglect the time.
+<img width="255" src="illustrations/descr_pics/Ra_U.jpg"/>
```

Comment:
There should be some separation between the section body and the image, otherwise I get this:
<img width="1467" height="364" alt="Screenshot_2025-11-15_19-19-59" src="https://github.com/user-attachments/assets/523f6a2b-6b5d-45ab-9572-71d55ef1d886" />